### DISCUSSION TITLE
Agent auth selected, but no running agent is detected Handshake failed: no matching key exchange algorithm

### DISCUSSION THREAD

**Discussion by rihanalkamim** [Eugeny/tabby#9431] [Q&A]
" SSH   !  Agent auth selected, but no running agent is detected
 X  Handshake failed: no matching key exchange algorithm"

How do fix it? 
Anybody can help me?

![image](https://github.com/Eugeny/tabby/assets/106154120/8a3df1b8-5277-4584-94c4-a59c5b1ca586)

**Comment by dk4633** [comment]
<img width="742" alt="1" src="https://github.com/Eugeny/tabby/assets/47392847/368a1f2b-9416-4f14-8419-f7ff96fde113">
<img width="831" alt="2" src="https://github.com/Eugeny/tabby/assets/47392847/d9de5460-6a28-43f2-9372-255498bf6890">

**Comment by hwp9527** [comment]
Has this problem been resolved? I encountered the same issue after upgrading to 1.0.226.
